src/static/fonts/fonts.css

```diff
@@ -8,15 +8,15 @@
   font-style: normal;
   font-weight: 100 900;
   font-display: swap;
-  src: url("/fonts/Inter/InterVariable.woff2?v=4.0") format("woff2");
+  src: url("/docs/fonts/Inter/InterVariable.woff2?v=4.0") format("woff2");
```

Comment:
Loading this statically allows us to define absolute paths and circumvent the webpack bundler